src/app/index.spec.tsx

```diff
@@ -119,7 +119,7 @@ describe('create app', () => {
         pathname: notFound.getUrl({url: 'url'}),
         replace: jest.fn(),
       };
-      delete window.location;
+      delete (window as any).location;
```

Comment:
It's a fix for `The operand of a 'delete' operator must be optional.`. Is it something we want to intentionally ignore?

Reply:
I think so